books/book.js

```diff
@@ -0,0 +1,40 @@
+let currentUrl = new URL(location.href); // creating new url that contains current url
+let searchParams = new URLSearchParams(currentUrl.search); //  searching in current  url params
+let currentUrlId = searchParams.get("id"); //  getting current url id// Fetching book when page is loaded
```

Comment:
A better name for this variable would be just `bookId`

Reply:
This is fixed as well:)